Modules/Map/QuestieMap.lua

```diff
@@ -378,17 +358,18 @@ function QuestieMap:ShowNPC(npcID, icon, scale, title, body, disableShiftToRemov
     data.IsObjectiveNote = false
     data.ManualTooltipData = {}
     data.ManualTooltipData.Title = title or (npc.name .. " (" .. l10n("NPC") .. ")")
+    local WeaponMasterSkills = require("WeaponMasterSkills")
     if Questie.IsTBC or Questie.IsWotlk then
         -- Blood Elf Starting Area Weapon Trainers
-        weaponMasterSkills[16621] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
-        weaponMasterSkills[17005] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
+        WeaponMasterSkills.data[16621] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
+        WeaponMasterSkills.data[17005] = {"Bows", "Daggers", "One-Handed Swords", "Polearms", "Thrown", "Two-Handed Swords"}
         -- Draenei Starting Area Weapon Trainers
-        weaponMasterSkills[16773] = {"Crossbows", "Daggers", "One-Handed Maces", "One-Handed Swords", "Two-Handed Maces", "Two-Handed Swords"}
+        WeaponMasterSkills.data[16773] = {"Crossbows", "Daggers", "One-Handed Maces", "One-Handed Swords", "Two-Handed Maces", "Two-Handed Swords"}
     end
```

Comment:
This should also be moved to the `WeaponMasterSkill` module.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5568/commits/2beca70ea6220a1102f0b8fc9057a3b3afd992c4